set.mm

```diff
@@ -704635,6 +704646,264 @@ exists a proof for (a and b).  (Contributed by Jarvin Udandy,
 $)
 $( Skip $[ set-main.mm $] $)
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  General auxiliary theorems (1)
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  The empty set - extension
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d X y $.
+    $( The empty set is the one and only set which is a subset of every set.
+       (Contributed by AV, 24-Aug-2022.) $)
+    al0ssb $p |- ( A. y X C_ y <-> X = (/) ) $=
+      ( cv wss wal c0 wceq 0ex sseq2 ss0b syl6bb 0ss ax-gen sseq1 albidv mpbiri
+      spcv impbii ) BACZDZAEZBFGZTUBAFHSFGTBFDUBSFBIBJKQUBUAFSDZAEUCASLMUBTUCAB
+      FSNOPR $.
+  $}
```

Comment:
It looks like this could be moved to Main (now or in a later PR).

Reply:
Do you think we can shorten sufficiently many proofs by it, or only because it is interesting?

Reply:
Only on its own interest. But this can be done (if you wish so) in a later PR.

Reply:
OK, I have moved it to main.